src/context.rs

```diff
@@ -176,6 +176,24 @@ pub struct Context {
 /// underlying solver (via
 /// [`ContextBuilder::solver`][crate::ContextBuilder::solver]).
 impl Context {
+    /// Directly send a command to the solver.
+    pub fn send(&mut self, cmd: SExpr) -> io::Result<()> {
+        let solver = self
+            .solver
+            .as_mut()
+            .expect("send requires a running solver");
+        solver.send(&self.arena, cmd)
+    }
+
+    /// Directly receive a response from the solver.
+    pub fn recv(&mut self) -> io::Result<SExpr> {
+        let solver = self
+            .solver
+            .as_mut()
+            .expect("recv requires a running solver");
+        solver.recv(&self.arena)
+    }
```

Comment:
Mind adding a `raw_` prefix (or something, open to bikeshedding) and a note in the doc comment to signify that these are low-level, "raw" APIs, `check`/`set_option`/etc. should generally be preferred, and are only intended as an escape hatch for solver-specific commands that this crate doesn't have built-in support for?

Reply:
Done! I also moved these calls to the bottom of the `impl` just so that they aren't at the top of `cargo doc`.